.github/workflows/pull-request-review.yml

```diff
@@ -97,6 +97,17 @@ jobs:
           This will be resolved by the Yivi team when your changes have been approved. You don't have to deal with this yourself."
         working-directory: ./base
 
+      - name: Write review comment if maintainers have no access to sign the changes
```

Comment:
no _write_ access ?

Reply:
Hmm yeah, but then I have to change it in all other repos too and it's only a text that's visible for us when the pipeline runs for documentation.